udata/api/__init__.py

```diff
@@ -9,9 +10,9 @@
 from flask import (
     current_app, g, request, url_for, json, make_response, redirect, Blueprint
 )
-from flask_restplus import Api, Resource, inputs, cors
+from flask_restplus import Api, Resource, cors
 
-from udata import search, theme, tracking
+from udata import search, theme, tracking  # NOQA: search must be imported.
```

Comment:
I don't know exactly, I just tried to remove it and it wasn't working anymore so I prefer it to be explicit :)